docs/extensions/ListViewExtensions.md

```diff
@@ -7,72 +7,128 @@ keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, ListViewBase, ext
 
 # ListViewExtensions
 
-ListViewExtensions extensions provide a lightweight way to extend every control that inherits the [ListViewBase](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.listviewbase) class with attached properties.
+[ListViewBaseExtensions](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.extensions.listviewbase) provide a lightweight way to extend every control that inherits the [ListViewBase](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ListViewBase) class with attached properties.
 
-##### AlternateColor
+## AlternateColor extentions
 
 The AlternateColor property provides a way to assign a background color to every other item.
 
 > [!WARNING]
 The [ContainerContentChanging](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.listviewbase#Windows_UI_Xaml_Controls_ListViewBase_ContainerContentChanging) event used for this extension to work, will not be raised when the ItemsPanel is replaced with another type of panel than ItemsStackPanel or ItemsWrapGrid. 
 
-## Example
+### Syntax
 
 ```xaml
+<Page ...
+     xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI.Extensions">
+
 <ListView
-    extensions:ListViewExtensions.AlternateColor="Silver"
+    extensions:ListViewBase.AlternateColor="Silver"
```

Comment:
This should be ListViewExtensions as ListViewBase is deprecated and will be removed